### PR TITLE
Expand README instruction

### PR DESCRIPTION
**Pull request by chuck-h**
Add explicit information about starting a local test net and creating the owner account.

### CODE REVIEW

File: include/seeds.accounts.hpp
Comment (n13) [inline include/seeds.accounts.hpp]: These changes are unrelated and should not be here

Ways to fix
A - merge master into this branch, should fix it , but if not
B - create new branch off master, only add readme updates, nothing else, create new PR

**Review by n13 — CHANGES_REQUESTED** [review]
Requesting only README file updated, no other files.

File: include/seeds.accounts.hpp
Comment (chuck-h) [reply include/seeds.accounts.hpp]: updated merge

**Review by n13 — APPROVED** [review]

**Comment by n13** [comment]
Oh - this is in your repo - need to have this PR on the main seeds smart contracts repo

**Comment by n13** [comment]
I can't merge this! As it is targeting your own repo, I think

It should be targeting the fork / upstream repo?

**Comment by n13** [comment]
And I don't have write access here so can't change it - waiting for @chuck-h

**Pull request closed without merging**
